docs/developer/process/agent-release/pre-release.md

```diff
@@ -32,6 +32,8 @@ all integrations with pending changes then branch off.
 
 1. Make a pull request to release [any new integrations](../integration-release.md#new-integrations), then merge it and pull `master`
 1. Make a pull request to release [all changed integrations](../integration-release.md#bulk-releases), then merge it and pull `master`
+    * Get 2+ thorough reviews on the changelogs. Titles should have good copy, and appropriate SemVer levels (e.g. `Changed` entries must refer to breaking changes only).
```

Comment:
copy?

Reply:
I mean copywriting, like syntax / grammar / typos

Reply:
We can probably drop this and just go for…

```suggestion
    * Get 2+ thorough reviews on the changelogs. Entries have appropriate SemVer levels (e.g. `Changed` entries must refer to breaking changes only).
```

WDYT?

Reply:
👍 & link to https://datadoghq.dev/integrations-core/guidelines/pr/